### DISCUSSION TITLE
libpng warning: iCCP: known incorrect sRGB profil

### DISCUSSION THREAD

**Discussion by yeuei** [israel-dryer/ttkbootstrap#559] [Q&A]
When I export the my own themes,the terminal have these warning "libpng warning: iCCP: known incorrect sRGB profil".And, I have no idea about how to use my own themes.

**Comment by TheDemonGuard** [comment]
@yeuei 
This warning is not coming from TTKBootstrap and according to the following documentation,
it seems that starting from libpng 1.6, error/warning detection has improved.
This might be one of the reasons you are getting this warning message(s).
https://www.libpng.org/pub/png/libpng-manual.txt#:~:text=Error%20detection%20in%20some%20chunks%20has%20improved

Some of the possible workaround the Warning:
1) Ignore the Warning:
If the color accuracy is not critical for your application and the warning doesn't cause any functional issues,
you can often safely ignore it.

2) Fix the PNG File:
Use image processing libraries (e.g., Python's Pillow, OpenCV) to load and resave the image,
which can sometimes correct the profile.

> When I export the my own themes,the terminal have these warning "libpng warning: iCCP: known incorrect sRGB profil".And, I have no idea about how to use my own themes.
